pkg/views/server/apikey/notify.go

```diff
@@ -5,41 +5,29 @@ package apikey
 
 import (
 	"fmt"
-	"os"
 
+	"github.com/atotto/clipboard"
 	"github.com/charmbracelet/lipgloss"
 	"github.com/daytonaio/daytona/pkg/views"
-	"golang.org/x/term"
 )
 
-var minimumLayoutWidth = 80
-
 func Render(key, apiUrl string) {
 	var output string
 
-	terminalWidth, _, err := term.GetSize(int(os.Stdout.Fd()))
-	if err != nil {
-		fmt.Println("error: Unable to get terminal size")
-		return
-	}
-
 	output += fmt.Sprintf("%s %s", views.GetPropertyKey("Generated API key: "), key) + "\n\n"
 
-	output += "Make sure to copy it as you will not be able to see it again." + "\n\n"
-
-	output += views.SeparatorString + "\n\n"
-
 	output += "You can connect to the Daytona Server from a client machine by running:"
 
-	formattedCommand := lipgloss.NewStyle().Foreground(views.Green).Render(fmt.Sprintf("daytona profile add -a \\\n%s \\\n-k %s", apiUrl, key))
-	command := lipgloss.NewStyle().Foreground(views.Green).Render(fmt.Sprintf("daytona profile add -a %s -k %s", apiUrl, key))
+	views.RenderContainerLayout(views.GetInfoMessage(output))
+
+	command := fmt.Sprintf("daytona profile add -a %s -k %s", apiUrl, key)
+	fmt.Println(lipgloss.NewStyle().Padding(0).Foreground(views.Green).Render(command) + "\n\n")
```

Comment:
We can remove the two newlines at the end of this `fmt.Print` (before and after)

<img width="1837" alt="image" src="https://github.com/user-attachments/assets/73372e05-ff07-421e-baad-7cbaf4792d31">